frontend/packages/core/src/Resolver/input.tsx

```diff
@@ -55,6 +56,12 @@ interface SchemaResolverProps extends Pick<AccordionProps, "expanded" | "onClick
   submitHandler: any;
 }
 
+const SchemaDetails = styled(AccordionDetails)({
+  "> *": {
+    flex: "1 50%",
+  },
+});
```

Comment:
last q, why this as opposed to nesting a <Grid> in AccordionDetails

Reply:
We would have to know which breakpoints to apply to each child based on the total number instead of letting flex do it for us.